common/src/main/kotlin/entity/DiscordGuild.kt

```diff
@@ -176,9 +148,15 @@ class DiscordPartialGuild(
     val owner: OptionalBoolean = OptionalBoolean.Missing,
     val permissions: Optional<Permissions> = Optional.Missing(),
     val features: List<GuildFeature>,
-    @SerialName("welcome_screen")
-    val welcomeScreen: Optional<DiscordWelcomeScreen> = Optional.Missing()
-)
+    @SerialName("welcome_screen") val welcomeScreen: Optional<DiscordWelcomeScreen> = Optional.Missing(),
+    @SerialName("vanity_url_code") val vanityUrlCode: Optional<String?> = Optional.Missing(),
+    val description: Optional<String?> = Optional.Missing(),
+    val banner: Optional<String?> = Optional.Missing(),
+    val splash: Optional<String?> = Optional.Missing(),
+    @SerialName("nsfw_level") val nsfwLevel: Optional<NsfwLevel> = Optional.Missing(),
+    @SerialName("verification_level")
+    val verificationLevel: Optional<VerificationLevel> = Optional.Missing()
```

Comment:
same for partial guilds?

Reply:
But I added them
 https://github.com/kordlib/kord/pull/478/files/c599823687518aec6522c8203f5ee75c408e191b#diff-81b031b8fa9334a46827df60852dd1720abfbd54770d8c1b1ab62bbe1ce9ba0bR151-R158